packages/mui-base/src/TextareaAutosize/TextareaAutoSize.test.tsx

```diff
@@ -61,13 +70,15 @@ describe('<TextareaAutosize />', () => {
       clock.withFakeTimers();
 
       it('should handle the resize event', () => {
-        const { container } = render(<TextareaAutosize />);
-        const input = container.querySelector('textarea[aria-hidden=null]');
+        const { container } = render(<TextareaAutosize minRows={1} />);
+        const input: HTMLInputElement = container.querySelector(
+          'textarea[aria-hidden=null]',
+        ) as HTMLInputElement;
         const shadow = container.querySelector('textarea[aria-hidden=true]');
         expect(input.style).to.have.property('height', '');
         expect(input.style).to.have.property('overflow', '');
 
-        setLayout(input, shadow, {
+        setLayout(input as HTMLInputElement, shadow as HTMLTextAreaElement, {
```

Comment:
No need to cast `input` as it has the correct type.
Also, make sure that `shadow` has the correct type to avoid casting.